caliban_toolbox/utils/widget_utils.py

```diff
@@ -38,65 +33,11 @@
 import numpy as np
 import skimage as sk
 import os
-from imageio import imread
-
-from caliban_toolbox.utils.io_utils import get_img_names
 
-def choose_img(name, dirpath):
```

Comment:
My preference is to keep the widget_utils intact as we need them to bridge between our older, platform-dependent crowdsourcing work and our new Caliban-based methodology

Reply:
Okay, sounds good. I didn't realize there still outstanding jobs with old version.